### PR TITLE
Ci cd setup

### PR DESCRIPTION
**Pull request by buddiex**
was able to setup ci for now with GitHub actions
we We will work on cd later

the idea here is for a service to be tested on only when the files in its service dir changes

### CODE REVIEW

File: twitter-producer-svc/app.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+from kafka.errors import KafkaError
+from pprint import pprint
+
+KAFKA_SVC = 'kafka:29092'
+
+producer = KafkaProducer(bootstrap_servers=[KAFKA_SVC],
+              api_version=(7,1,0),
+              )
+
+# future = producer.send('my-topic', b'raw_bytes')
```
Comment (aayeni1) [inline twitter-producer-svc/app.py:12]: Is this block informational or do we plan to use in future ? for example, you already implemented line 22-25 in **on_send_success** function. A lot of similar instances of commented blocks, if they are indeed useful in future, consider including a line stating the future use case.

**Review by aayeni1 — DISMISSED** [review]
LGTM

File: twitter-producer-svc/app.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+# Block for 'synchronous' sends
```
Comment (aayeni1) [inline twitter-producer-svc/app.py]: Is this block (12-36) informational or do we plan to use in future ? for example, you already implemented line 22-25 in on_send_success function. A lot of similar instances of commented blocks, if they are indeed useful in future, consider including a line stating the future use case.

Comment (buddiex) [reply twitter-producer-svc/app.py]: Nothing in this file is useful for the current pipeline. I will take them out.

**Comment by King-Ob** [comment]
approv

**Review by King-Ob — DISMISSED** [review]
looks good

**Review by aayeni1 — APPROVED** [review]

**Review by noob-art — APPROVED** [review]

**Pull request merged**